runtime/pkg/duration/duration.go

```diff
@@ -36,15 +57,23 @@ var (
 func ParseISO8601(from string) (Duration, error) {
 	// Try parsing for "inf"
 	if infPattern.MatchString(from) {
-		return Duration{Inf: true}, nil
+		return StandardDuration{Inf: true}, nil
+	}
+
+	rillDur := strings.Replace(from, "rill-", "", 1)
+	if a, ok := daxToDateNotations[rillDur]; ok {
+		return ToDateDuration{anchor: a}, nil
+	}
+	if o, ok := daxOffsetNotations[rillDur]; ok {
+		return o, nil
 	}
```

Comment:
A brief comment explaining the `rill-` prefix here being used for dax would be helpful (future folks will lack that context)